spec/java_buildpack/buildpack_version_spec.rb

```diff
@@ -96,6 +96,28 @@
     expect(buildpack_version.to_hash).to eq({})
   end
 
+  it 'excludes remote string when remote and hash values from config/version.yml are empty',
+     configuration: { 'hash' => '', 'remote' => '', 'version' => 'test-version' } do
+    expect(buildpack_version.to_s).to eq('test-version')
+  end
+
+  it 'excludes include remote string when remote and hash values from config/version.yml are missing',
```

Comment:
Pretty sure that this should be simply `includes remote string when remote and ...`.  Can you please update?